css/styles.css

```diff
@@ -46,4 +48,60 @@ a {
   margin-top: 1rem;
   margin-bottom: 1rem;
   letter-spacing: 0.35rem;
+  cursor: pointer;
+}
+
+.habit-form {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.row.habit {
+  display: flex;
+  flex: 1;
+  overflow: auto;
+  padding-top: 3rem;
+}
+
+#enter-habit {
+  background-color: rgb(238, 110, 115, 0.3);
+  border-style: none;
+  border-radius: 0.2rem;
+  margin-right: 2rem;
+  padding: 0.5rem;
+  font-size: 18px;
```

Comment:
There's almost always better alternative to `px` for font sizes: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css

Reply:
Please convert this to `rem`.